data_model/src/expression.rs

```diff
@@ -775,8 +977,20 @@ impl From<ContainsAny> for ExpressionBox {
 
 /// Returns `true` if `left` operand is equal to the `right` operand.
 #[derive(
-    Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, IntoSchema, PartialOrd, Ord,
+    Debug,
+    Display,
+    Clone,
+    PartialEq,
+    Eq,
+    Decode,
+    Encode,
+    Deserialize,
+    Serialize,
+    IntoSchema,
+    PartialOrd,
+    Ord,
 )]
+#[display(fmt = "`{}` == `{}`", left, right)]
```

Comment:
```suggestion
#[display(fmt = "{} == {}", left, right)]
```

In general terms on both sides of `equals` are not parenthesised.